modules/learn/pages/services-and-indexes/indexes/storage-modes.adoc

```diff
@@ -0,0 +1,63 @@
+= Storage Settings
+:description: All indexes in Couchbase Capella use the Couchbase Plasma storage engine. 
+:page-topic-type: concept
+:page-aliases: indexes:storage-modes,understanding-couchbase:services-and-indexes/indexes/storage-modes,architecture:index-storage
+
+[abstract]
+{description}
+
+
+[#standard-index-storage]
+== Standard Index Storage
+
+In Couchbase Capella, _Standard_ is the storage-setting for Secondary Indexes: the indexes are saved on disk; in a disk-optimized format that uses both memory and disk for index-update and scanning.
+The performance of standard index storage depends on overall I/O performance.
+
+The standard global secondary index uses a B-Tree index and keeps the optimal working set of data in the buffer.
+This means the total size of the index can be much bigger than the amount of memory available in each index node.
+
+Standard index-storage is supported by the _Plasma_ storage engine.
```

Comment:
More editing to remove mention of "standard", because it's the only storage available

```suggestion
The performance of index storage depends on overall I/O performance.

The global secondary index uses a B-Tree index and keeps the optimal working set of data in the buffer.
This means the total size of the index can be much bigger than the amount of memory available in each index node.

Couchbase Capella index-storage is supported by the _Plasma_ storage engine.
```